File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfaceSalvaProvas.java
package tech.leonam.resultadodocertame.model.interfaces;

import android.content.Context;

import tech.leonam.resultadodocertame.model.entidade.ConfigProva;

public interface InterfaceSalvaProvas {
    boolean salvaProvas(Context context, ConfigProva configProva);
}

File: app/src/main/java/tech/leonam/resultadodocertame/model/interfaces/InterfaceSalvaProvas.kt
package tech.leonam.resultadodocertame.model.interfaces

import android.content.Context
import tech.leonam.resultadodocertame.modelView.entidade.ConfigProva

interface InterfaceSalvaProvas {
    @Throws(Exception::class)
    fun salvaProvas(context: Context?, configProva: ConfigProva?): Boolean
}